packages/SwingSet/test/test-controller.js

```diff
@@ -1,9 +1,7 @@
-/* global require __dirname */
 // eslint-disable-next-line import/order
+import { spawn } from 'child_process';
```

Comment:
Will this sequence of imports be stable under `yarn lint-fix`? That `import/order` override was meant to be glued to the `prepare-test-env-ava`, and I worry that without it, the prepare-test-env-ava will migrate forwards, beyond `hostStorage.js`.